eslint/babel-eslint-plugin-development/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@babel/eslint-plugin-development",
-  "version": "1.0.1",
-  "description": "A set of eslint rules to enforce best practices in the development of Babel plugins.",
+  "version": "0.0.0",
```

Comment:
@kaicataldo I'm going to revert this line because it breaks e2e Babel tests with Yarn 2.
Was there any reason to change it? (I don't remember)

Reply:
Hmm I think maybe that was a mistake. Sorry about that!